ai-service/tests/test_llm_and_search.py

```
@@ -3,74 +1,79 @@
-#     if test_collection.peek()["ids"]:
-#         test_collection.delete(ids=test_collection.peek()["ids"])
-#     question = "This code does not exist."
-#     question_embedding = embedder.embed_text(question)
-#     results = db.query_chunks(
-#         question_embedding,
-#     )
-#     # Should return an empty or placeholder result
-#     docs = results.get("documents")
-#     assert docs is not None and (docs[0] == [] or (docs[0] and docs[0][0] == ""))
 
+def test_db_search_no_results():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_db_search_no_results`

Add return type annotation: `None`

(ANN201)